yt/utilities/parallel_tools/tests/test_dask_helper.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import pytest
+
+from yt.config import ytcfg
+from yt.testing import assert_array_equal, assert_equal
+from yt.utilities.on_demand_imports import _dask as dask
+from yt.utilities.parallel_tools import dask_helper
+
+
+@pytest.fixture(autouse=True, scope="module")
+def with_dask_enabled():
+    # Fixture to set dask_enabled config to standard value then reset.
+    # ensures that these tests will work even if default value for dask_enabled
+    # changes while ensuring the config value gets reset to the default value
+    # after a test runs.
+
+    original_config = ytcfg.get("yt", "internals", "dask_enabled")
+    ytcfg.set("yt", "internals", "dask_enabled", False)
+
+    yield  # test will run here
+
+    ytcfg.set("yt", "internals", "dask_enabled", original_config)
+
+
+@pytest.mark.skipif(not dask.__is_available__, reason="requires dask")
```

Comment:
with `from yt.testing import requires_module_pytest` :)
```suggestion
@requires_module_pytest("dask")
```